### ISSUE TITLE
Error: Failed to load config "developit" to extend from.

### ISSUE THREAD

**Issue by Luferov** [nuxt/eslint-config#155]
.eslintrc.js
```js
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true
  },
  extends: [
    '@nuxtjs/eslint-config-typescript',
    'plugin:nuxt/recommended'
  ]
}
```

package.json

```json
{
  "name": "client",
  "version": "1.1.0",
  "private": true,
  "scripts": {
    "dev": "nuxt",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "graphql-codegen --config codegen.yml && node ./schemaQuery.js",
    "generate-server": "graphql-codegen --config codegen_server.yml && node ./schemaQuery.js",
    "generate-nuxt": "nuxt generate",
    "lint:js": "eslint --ext .js,.vue --ignore-path .gitignore .",
    "lint": "yarn lint:js",
    "test": "jest"
  },
  "dependencies": {
    "@nuxt/types": "^2.15.4",
    "@nuxtjs/apollo": "^4.0.1-rc.5",
    "@nuxtjs/pwa": "^3.3.5",
    "@nuxtjs/tailwindcss": "^4.0.3",
    "@nuxtjs/vuetify": "^2.0.0-beta.2",
    "graphql-tag": "^2.12.3",
    "nuxt": "^2.15.4",
    "vee-validate": "^3.4.5",
    "vue-server-renderer": "^2.6.12",
    "vue-template-compiler": "^2.6.12"
  },
  "devDependencies": {
    "@babel/plugin-proposal-optional-chaining": "^7.13.12",
    "@graphql-codegen/cli": "^1.21.4",
    "@graphql-codegen/schema-ast": "^1.18.2",
    "@graphql-codegen/typescript": "^1.22.0",
    "@graphql-codegen/typescript-operations": "^1.17.16",
    "@nuxt/typescript-build": "^2.1.0",
    "@nuxtjs/eslint-config": "^6.0.0",
    "@nuxtjs/eslint-config-typescript": "^6.0.0",
    "@nuxtjs/eslint-module": "^3.0.2",
    "@vue/test-utils": "^1.2.0",
    "babel-core": "^7.0.0-bridge.0",
    "babel-eslint": "^10.1.0",
    "babel-jest": "^26.6.3",
    "core-js": "^3.11.2",
    "eslint": "^7.16.0",
    "eslint-plugin-nuxt": "^2.0.0",
    "jest": "^26.5.0",
    "js-cookie": "^2.2.1",
    "nuxt-i18n": "^6.26.0",
    "postcss": "^8.2.9",
    "pug": "^3.0.2",
    "pug-plain-loader": "^1.1.0",
    "ts-jest": "^26.4.1",
    "vue-class-component": "^7.2.6",
    "vue-jest": "^3.0.4",
    "vue-property-decorator": "^9.1.2",
    "vuetify": "^2.4.11"
  }
}
```
nuxt.config.ts
```ts
import colors from 'vuetify/es5/util/colors'

export default {
  telemetry: true,
  /*
  ** Nuxt rendering mode
  ** See https://nuxtjs.org/api/configuration-mode
  */
  ssr: true,
  /*
  ** Nuxt target
  ** See https://nuxtjs.org/api/configuration-target
  */
  target: 'server',
  loading: { color: '#413dff' },
  // Global CSS (https://go.nuxtjs.dev/config-css)
  css: [
  ],

  // Plugins to run before rendering page (https://go.nuxtjs.dev/config-plugins)
  plugins: [
    '~/plugins/apollo-init',
    '~/plugins/filters',
    '~/plugins/utils',
    '~/plugins/vuetify'
  ],

  // Auto import components (https://go.nuxtjs.dev/config-components)
  components: true,

  // Modules for dev and build (recommended) (https://go.nuxtjs.dev/config-modules)
  buildModules: [
    '@nuxt/typescript-build',
    '@nuxtjs/tailwindcss',
    '@nuxtjs/vuetify',
    ['nuxt-i18n', i18n || {
      lazy: true,
      langDir: 'lang/',
      defaultLocale: 'en',
      locales: [
        { code: 'en', file: 'en.ts' }
      ]
    }]
  ],
  // Modules (https://go.nuxtjs.dev/config-modules)
  modules: [
    '@nuxtjs/apollo',
  ],

  // Content module configuration (https://go.nuxtjs.dev/config-content)
  content: {},
  apollo: {
    includeNodeModules: true,
    clientConfigs: {
      default: '~/plugins/apollo.ts'
    }
  },
  router: {
    middleware: ['check-auth']
  },
  /*
  ** vuetify module configuration
  ** https://github.com/nuxt-community/vuetify-module
  */
  vuetify: {
    treeShake: true,
    theme: {
      dark: true,
      themes: {
        dark: {
          primary: colors.blue.darken2,
          accent: colors.grey.darken3,
          secondary: colors.amber.darken3,
          info: colors.teal.lighten1,
          warning: colors.amber.base,
          error: colors.deepOrange.accent4,
          success: colors.green.accent3
        }
      }
    }
  },
  typescript: {
    typeCheck: {
      eslint: {
        files: './**/*.{ts,js,vue}'
      }
    }
  },
  build: {
    analyze: process.client,
    parallel: true,
    transpile: [
      'vee-validate/dist/rules'
    ],
    babel: {
      plugins: [['@babel/plugin-proposal-private-methods', { loose: true }]]
    }
  }
}
```
node: v14.16.1
typesctipt: 4.2.4

When `yarn run dev`, I get the following error:
![image](https://user-images.githubusercontent.com/26242554/117017071-fd37e680-acfb-11eb-8b3b-d23227758c3f.png)

ERROR  Error: Failed to load config "developit" to extend from.                                                                                                                                                  17:11:36
Referenced from: /home/victor/PycharmProjects/eleden/client/node_modules/@babel/preset-modules/package.json

  Referenced from: node_modules/@babel/preset-modules/package.json
  at configInvalidError (node_modules/@eslint/eslintrc/lib/config-array-factory.js:290:9)
  at ConfigArrayFactory._loadExtendedShareableConfig (node_modules/@eslint/eslintrc/lib/config-array-factory.js:883:23)
  at ConfigArrayFactory._loadExtends (node_modules/@eslint/eslintrc/lib/config-array-factory.js:781:25)
  at ConfigArrayFactory._normalizeObjectConfigDataBody (node_modules/@eslint/eslintrc/lib/config-array-factory.js:720:25)
  at _normalizeObjectConfigDataBody.next (<anonymous>)
  at ConfigArrayFactory._normalizeObjectConfigData (node_modules/@eslint/eslintrc/lib/config-array-factory.js:665:20)
  at _normalizeObjectConfigData.next (<anonymous>)
  at ConfigArrayFactory.loadInDirectory (node_modules/@eslint/eslintrc/lib/config-array-factory.js:511:28)
  at CascadingConfigArrayFactory._loadConfigInAncestors (node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:379:46)
  at CascadingConfigArrayFactory._loadConfigInAncestors (node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:398:20)

Could you tell me, please, what could be the problem?

**Comment by rob-tech** [comment]
Same issue! nuxt: 2.15.4,  node: v14. This started happening once I added the babel plugin: [['@babel/plugin-proposal-private-methods', { loose: true }]] to resolve the following issue: https://github.com/nuxt/nuxt.js/pull/9226

**Comment by joaovinicius** [comment]
It works for me

In nuxt.config.js 

```
build: {
    babel: { babelrc: true },
}
```

In .babelrc

```
{
  "plugins": [
    ["@babel/plugin-proposal-private-methods", { "loose": true }]
  ],
  "env": {
    "test": {
      "presets": [
        [
          "@babel/preset-env",
          {
            "targets": {
              "node": "current"
            }
          }
        ]
      ]
    }
  }
}
```

**Comment by pi0** [comment]
Hi! I'm closing this issue since seems stalled. Please create a new one if still relevant 🙏🏼
